lib/internal/Magento/Framework/Validator/Factory.php

```diff
@@ -6,65 +6,50 @@
 
 namespace Magento\Framework\Validator;
 
-use Magento\Framework\Cache\FrontendInterface;
+use Magento\Framework\Module\Dir\Reader;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\Phrase;
+use Magento\Framework\Validator;
 
+/**
+ * Factory for \Magento\Framework\Validator and \Magento\Framework\Validator\Builder.
+ */
 class Factory
 {
-    /** cache key */
-    const CACHE_KEY = __CLASS__;
```

Comment:
Unfortunately, we are not allowed to remove the constants according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Just mark it as deprecated

Reply:
Thanks for your feedback. I addressed this in my latest commit.